public/403-no-access.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>You do not have access to this page (403)</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <script src="status_code.js"></script>
+  <link rel="stylesheet" href="/status_code.css"/>
+</head>
+
+<body class="rails-default-error-page">
+  <!-- This file lives in public/403_no_access.html -->
+  <div class="dialog">
+    <div>
+      <h1>Welcome to the Accessable Van Service website</h1>
+      <h2>You are not allowed on this page.</h2>
+      <p>Some possible reasons:
+        <ul>
+          <li>
+            You are attempting to login with a subsidiary account. 
+            Role subsidiary accounts cannout be used to access this site. 
+            Please login with your primary NetID. For assistance, contact 
+            <a href="mailto:transit-it@admin.umass.edu">transit-it@admin.umass.edu</a>
+          </li>
+          <li>
+            You are a current or future passenger. In this case, you may be interested in our 
+            <a href="/passengers/brochure">Brochure.</a>
+          </li>
+          <li>
+            You are an AVS employee and believe you should have access. 
+            In this case, please contact AVS staff for assistance.
+          </li>
+        </ul>
+      </p>
+      <p>
+        <form method="post" action="/logout">
+          <a href="#" class="logout-link">Logout</a>
+        </form>
+      </p>
```

Comment:
```suggestion
      <h1>You do not have access to this page.</h1>
      <p>Please submit feedback if you believe this is a mistake.</p>
      <nav>
        <a href="/">Home</a>
        <a href="/feedback/new">Feedback</a>
        <form method="post" action="/logout">
          <a href="#" class="logout-link">Logout</a>
        </form>
      </nav>
```
Make this a little more concise/add a home and feedback link

Reply:
See below for styling for the nav